src/PythonMigrationViewExtension/GraphPythonDependencies.cs

```diff
@@ -18,10 +21,49 @@ internal GraphPythonDependencies(ViewLoadedParams viewLoadedParams)
         internal bool ContainsIronPythonDependencies()
         {
             var workspace = ViewLoaded.CurrentWorkspaceModel;
+
             if (workspace == null)
                 throw new ArgumentNullException(nameof(workspace));
 
-            return workspace.Nodes.Any(n => IsIronPythonNode(n));
+            if (workspace.Nodes.Any(n => IsIronPythonNode(n)))
+            {
+                return true;
+            }
+
+            // Check if any of the custom nodes has IronPython dependencies in it. 
+            var customNodeManager = ViewLoaded.StartupParams.CustomNodeManager;
+            var customNodes = workspace.Nodes.OfType<Function>();
+
+            if (CustomNodesContainIronPythonDependencies(customNodes, customNodeManager))
+            {
+                return true;
+            }
+
+            return false;
```

Comment:
You can directly return `CustomNodesContainIronPythonDependencies(customNodes, customNodeManager)` here.

Reply:
Actually, if it returns false we don't want to return it because we needs to continue checking for other custom nodes till it finds the first one.

Reply:
The above single liner is effectively doing the same thing as the `if-else`.

Reply:
Ah, sorry, I thought you were referring to the CustomNodesContainIronPythonDependencies() call in the recursive function. Changed it accordingly.